RankSys-diversity/src/main/java/es/uam/eps/ir/ranksys/diversity/intentaware/AspectModel.java

```diff
@@ -89,11 +88,10 @@ public double pf_u(F f) {
     /**
      * Item aspect model for {@link AspectModel}.
      *
-     * @param <U> user type
      * @param <I> item type
      * @param <F> aspect type
      */
-    public interface ItemAspectModel<U, I, F> extends Model<U> {
```

Comment:
Doesn't it remove the caching capability?
That would make it slightly inconsistent I think - if caching is used, UAM would be cached and IAM not.
Now I'm thinking that maybe caching could be removed from the Model and if one would like to cache models, could decorate that with e.g. CachedModel.